src/NzbDrone.Core/Datastore/Migration/210_add_monitored_seasons_filter.cs

```diff
@@ -0,0 +1,76 @@
+using System.Collections.Generic;
+using System.Data;
+using Dapper;
+using FluentMigrator;
+using Newtonsoft.Json.Linq;
+using NzbDrone.Common.Serializer;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(210)]
+    public class add_monitored_seasons_filter : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Execute.WithConnection(ChangeHasUnmonitoredSeason);
+        }
+
+        private void ChangeHasUnmonitoredSeason(IDbConnection conn, IDbTransaction tran)
+        {
+            var updated = new List<object>();
+            using (var getUnmonitoredSeasonFilter = conn.CreateCommand())
+            {
+                getUnmonitoredSeasonFilter.Transaction = tran;
+                getUnmonitoredSeasonFilter.CommandText = "SELECT \"Id\", \"Filters\" FROM \"CustomFilters\"";
```

Comment:
No need to process Custom Filters for things that won't have this particular filter:

```suggestion
                getUnmonitoredSeasonFilter.CommandText = "SELECT \"Id\", \"Filters\" FROM \"CustomFilters\" WHERE \"Type\" = 'series'";
```